zipline/finance/trading.py

```diff
@@ -69,7 +70,7 @@ def __init__(
         bm_symbol='^GSPC',
         exchange_tz="US/Eastern",
         max_date=None,
-        env_trading_calendar=tradingcalendar,
+        env_trading_calendar=NYSEExchangeCalendar,
```

Comment:
Why pass in a calendar class, rather than an instantiated Calendar object? Doing the construction internally precludes us from giving the calendar arguments, if that is something that we ever want to do.

That may not be worth worrying about, though, if the eventual goal is to have all of the Calendar work removed from this class anyway.